src/App.js

```diff
@@ -1,23 +1,13 @@
-import logo from './logo.svg';
+/* eslint-disable no-unused-vars */
+
+import React, { Component } from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] The `App.js` component should be class-based. As the purpose of this milestone is to learn and implement the class-based component.